common/xrdp_constants.h

```diff
@@ -38,6 +38,7 @@
  ******************************************************************************/
 
 #define INFO_CLIENT_NAME_BYTES  32
+#define INFO_CLIENT_MAX_CB_LEN  512
```

Comment:
This is an interesting one. The value is directly derived from [MS-RDPBCGR] TS_INFO_PACKET(2.2.1.11.1.1), but there's no actual name for this length (as far as I can tell). It might be better placed in common/ms-rdpbcgr.h than here, but given the lack of an actual definition on the docs I don't really have string feelings either way. If it stays here, a reference to where it's from would be useful - either [MS-RDBBCGR] of elsewhere.

Reply:
Can do that, it is also in libscp_v0 btw (definition seem to be all over the place or hardcoded :/).